subprojects/plugins/src/main/java/org/gradle/api/plugins/jvm/internal/DefaultJvmTestSuite.java

```diff
@@ -384,4 +325,20 @@ public TaskDependency getBuildDependencies() {
             getTargets().forEach(context::add);
         });
     }
+
+    private class ToolchainFactory {
```

Comment:
💅 Shouldn't this just be `static` and inject the `ObjectFactory` and `InstantiatorFactory` in a constructor?

Reply:
Yeah, I think there was something about it in an earlier incarnation that needed to be non-static, but you're right, it should be static.